src/main/java/gov/nasa/pds/tools/util/FileSizesUtil.java

```diff
@@ -33,10 +33,19 @@ public static long getExternalFilesize(URL url) throws Exception {
 
       if (url.getProtocol().equals("file")) {
             long fileSize = -1;
-            LOG.debug("getExternalFilesize:url,fileSize {}",url,fileSize);
-            File file = new File(url.getPath());
+            LOG.debug("getExternalFilesize:url,fileSize {},{}",url,fileSize);
+            LOG.debug("getExternalFilesize:afor:path [{}]",url.getPath());
+            // For some strange reason, the File class does not handle well with the "%20" in the name.  It needs to be an actual space.
+            String actualPath = url.getPath().replace("%20"," ");  // Replace the "%20" with one space " "
```

Comment:
@qchaupds what do we think about just translating the path using [URLDecoder](https://docs.oracle.com/javase/7/docs/api/java/net/URLDecoder.html) (or some equivalent?). It seems like since we are treating these file paths as URIs, it is just being URL encoded, so we should just decode when we try to read it as a local file system path. does that make sense? if not, definitely let me know.

Reply:
@jordanpadams Changing the code to use URLDecoder.encode() seems to work.  The run produces the correct file size.  Code has been checked back in.  Please have another look.

{pds-dev3.jpl.nasa.gov}/home/qchau/sandbox/validate 108 % ls -l /home/qchau/sandbox/validate/src/test/resources/github381/a\ test/minmax-error.csv
-rw-r--r-- 1 qchau pds 7833375 Aug  4 14:40 /home/qchau/sandbox/validate/src/test/resources/github381/a test/minmax-error.csv


{pds-dev3.jpl.nasa.gov}/home/qchau/sandbox/validate 106 % validate -R pds4.label --skip-context-validation --skip-context-reference-check -r report_github381_label_valid_4.json   -s json -t  src/test/resources/github381/a\ test/minmax-error.xml  >& t5
{pds-dev3.jpl.nasa.gov}/home/qchau/sandbox/validate 107 % grep FileSizesUtil t5
[main] DEBUG gov.nasa.pds.tools.util.FileSizesUtil - getExternalFilesize:url file:/home/qchau/sandbox/validate/src/test/resources/github381/a%20test/minmax-error.csv
[main] DEBUG gov.nasa.pds.tools.util.FileSizesUtil - getExternalFilesize:url,fileSize file:/home/qchau/sandbox/validate/src/test/resources/github381/a%20test/minmax-error.csv,-1
**[main] DEBUG gov.nasa.pds.tools.util.FileSizesUtil - getExternalFilesize:afor:path [/home/qchau/sandbox/validate/src/test/resources/github381/a%20test/minmax-error.csv]
[main] DEBUG gov.nasa.pds.tools.util.FileSizesUtil - getExternalFilesize:after:path [/home/qchau/sandbox/validate/src/test/resources/github381/a test/minmax-error.csv]**
[main] DEBUG gov.nasa.pds.tools.util.FileSizesUtil - getExternalFilesize:File exist [/home/qchau/sandbox/validate/src/test/resources/github381/a test/minmax-error.csv]
[main] DEBUG gov.nasa.pds.tools.util.FileSizesUtil - getExternalFilesize:url,fileSize file:/home/qchau/sandbox/validate/src/test/resources/github381/a%20test/minmax-error.csv,7833375

The regression test ran OK too.

[INFO] Tests run: 118, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 120.349 s - in cucumber.CucumberTest
[INFO] 
[INFO] Results:
[INFO] 
[WARNING] Tests run: 119, Failures: 0, Errors: 0, Skipped: 1
[INFO] 
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  02:18 min
[INFO] Finished at: 2021-08-09T08:52:48-07:00
[INFO] ------------------------------------------------------------------------